src/test/java/org/openrewrite/java/testing/junit5/CsvSourceToValueSourceTest.java

```diff
@@ -0,0 +1,457 @@
+                      System.out.println(value);
+                  }
+              }
+              """,
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.provider.ValueSource;
+
+              class TestClass {
+                  @ParameterizedTest
+                  @ValueSource(strings = {"test1", "test2", "test3"})
+                  void testWithValueAttribute(String value) {
+                      System.out.println(value);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void replaceCsvSourceWithChars() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.provider.CsvSource;
+
+              class TestClass {
+                  @ParameterizedTest
+                  @CsvSource({"a", "b", "c"})
+                  void testWithChars(char letter) {
+                      System.out.println(letter);
+                  }
+              }
+              """,
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.provider.ValueSource;
+
+              class TestClass {
+                  @ParameterizedTest
+                  @ValueSource(chars = {'a', 'b', 'c'})
+                  void testWithChars(char letter) {
+                      System.out.println(letter);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void doNotReplaceWhenArgumentsAccessorIsUsed() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.aggregator.ArgumentsAccessor;
+              import org.junit.jupiter.params.provider.CsvSource;
+
+              class TestClass {
+                  @ParameterizedTest
+                  @CsvSource({"apple, 1", "banana, 2", "cherry, 3"})
+                  void testWithArgumentsAccessor(ArgumentsAccessor args) {
+                      String fruit = args.getString(0);
+                      int count = args.getInteger(1);
+                      System.out.println(fruit + ": " + count);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void doNotReplaceWhenAggregateWithIsUsed() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.aggregator.AggregateWith;
+              import org.junit.jupiter.params.provider.CsvSource;
+
+              class TestClass {
+                  @ParameterizedTest
+                  @CsvSource({"John, Doe", "Jane, Smith"})
```

Comment:
Also here I'd favor a single line:

```suggestion
                  @CsvSource("John, Doe")
```